static/js/publisher/form/fileInput.js

```diff
@@ -13,11 +13,19 @@ class FileInput extends React.Component {
     this.keyboardEventHandler = this.keyboardEventHandler.bind(this);
   }
 
-  componentDidUpdate() {
-    const { active, clear } = this.props;
-    if (active && clear && this.input) {
-      this.input.value = "";
+  componentDidMount() {
+    const { restrictions, inputName } = this.props;
+
+    this.input = document.createElement("input");
```

Comment:
Is this needed or would removing `clear` be enough of a fix?

Sure, it kind of prevents React from accidentally rendering new input fields on render, but I guess that React is smart enough to not rerender DOM when not needed (in this case if restrictions and inputName don't change)?

Reply:
Maybe it is needed... I did a quick test with just old `componentDidUpdate` removed to prevent from clearing the input and it worked better, but I still had cases of missing uploads when adding/removing many screenshots at once in crazy order.

But please add  a comment explaining why we add input manually here instead of properly rendering it with React.